bitnami/fluentd/templates/forwarder-clusterrole.yaml

```
@@ -28,7 +28,7 @@ rules:
     resources:
       - "podsecuritypolicies"
     resourceNames:
-      - {{ include "common.names.fullname" . }}-forwarder
+      - {{ printf "%s-%s" (include "common.names.fullname.namespace" .) "-forwarder" }}
```

Small typo, you would be adding an extra dash:

```suggestion
      - {{ printf "%s-%s" (include "common.names.fullname.namespace" .) "forwarder" }}
```

---

Good catch! I've fixed it now.